webapp/packages/plugin-data-viewer/src/DatabaseDataModel/DataContext/DATA_CONTEXT_DV_PRESENTATION_ACTIONS.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createDataContext } from '@cloudbeaver/core-data-context';
+
+import type { IDataPresentationActions } from '../../TableViewer/IDataPresentationActions.js';
+import type { IResultSetElementKey } from '../Actions/ResultSet/IResultSetDataKey.js';
+
+export const DATA_CONTEXT_DV_PRESENTATION_ACTIONS = createDataContext<IDataPresentationActions<IResultSetElementKey>>(
+  'data-viewer-database-presentation-actions',
+);
```

Comment:
should be in the view part cuz using `IDataPresentationActions` (view specific actions)